assets/js/common/ActivityLogsConfig/ActivityLogsConfig.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { noop } from 'lodash';
+
+import Button from '@common/Button';
+
+/**
+ * Simplistic utility to transform a time interval in a readable string
+ */
+function TimeLabel({ time }) {
+  if (typeof time === 'string') {
+    return time;
+  }
+  if (
+    typeof time === 'object' &&
+    time !== null &&
+    'value' in time &&
+    'unit' in time
+  ) {
+    return `${time.value} ${time.unit}${time.value > 1 ? 's' : ''}`;
```

Comment:
I see what you want to do and I appreciate, however we might have an issue here because the backend would return `days`, `weeks`, `months`, `years`, so we might end up with showing an extra trailing `s` as in `dayss`.

I guess our options are:
- we accept `1 days` and avoid the transformation
- we change the backend to return singulars `day`, `week`, `month`, `year`
- we map in the frontend the plurals returned by the backend to either singulars or plurals given the context

I don't like the third option with double mapping.
I could live with option 1 but we'd miss the opportunity to provide _correct_ copy.
Option 2 sounds quite reasonable, then up to you whether using a proper inflector or just adding a trailing `s`.

What do you think?

Reply:
I think I'll go for 1. No code, no problems

Reply:
I guess we can afford singularizing on the backend https://github.com/trento-project/web/pull/2709.
You can safaly assume the API would return singulars  `day`, `week`, `month`, `year`

Reply:
fixed in https://github.com/trento-project/web/pull/2706/commits/f6c4315e173e09a5dc71cba162b6e3df8669ac18